EMSBackendBridgeBundle/DependencyInjection/EMSBackendBridgeExtension.php

```diff
@@ -118,12 +118,22 @@ private function processClearCache(ContainerBuilder $container, $domain, array $
     private function defineElasticsearchClient(ContainerBuilder $container, $name, array $options)
     {
         $definition = new Definition(Client::class);
+        $config = ['hosts' => $options['hosts']];
+
+        foreach ($options['hosts'] as $host){
+            if((substr($host, 0, 8) === 'https://')){
+                $caBundle = \Composer\CaBundle\CaBundle::getBundledCaBundlePath();
```

Comment:
\Composer\CaBundle\CaBundle import class and use "CaBundle"

Reply:
It has been fixed